csharp/NedapRetailApiClient/Nedap.Retail.Api/Erp.V1/ErpV1Endpoint.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Nedap.Retail.Api.OAuth;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+
+namespace Nedap.Retail.Api.Erp.V1
+{
+    /// <summary>
+    /// Erp V1 API endpoint
+    /// </summary>
+    public class ErpV1Endpoint
+    {
+        private ApiCaller apiCaller;
+
+        internal ErpV1Endpoint(ApiCaller apiCaller)
+        {
+            this.apiCaller = apiCaller;
+        }
+
+        /// <summary>
+        /// Capture the ERP stock for a specific location on a specific time. The ERP stock is described in GTIN + quantity.
+        /// </summary>
+        /// <param name="location">The location for which the stock is defined</param>
+        /// <param name="eventTime">Time of export from ERP</param>
+        /// <param name="quantityList">Complete stock summary for this location at this eventTime</param>
+        /// <param name="externRef">Optional, external reference that can be used to identify this ERP stock count</param>
+        /// <returns>Stock ID of the saved ERP stock</returns>
+        public string StockCapture(string location, DateTime eventTime, List<GtinQuantity> quantityList, String externRef)
+        {
+            CaptureRequest request = new CaptureRequest();
```

Comment:
Why not check for required parameters here and in other methods of this endpoint?